tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/AspireEFPostgreSqlExtensionsTests.cs

```diff
@@ -109,8 +119,235 @@ public void CanConfigureDbContextOptions()
         Assert.NotNull(extension.ExecutionStrategyFactory);
         var executionStrategy = extension.ExecutionStrategyFactory(new ExecutionStrategyDependencies(new CurrentDbContext(context), context.Options, null!));
         var retryStrategy = Assert.IsType<NpgsqlRetryingExecutionStrategy>(executionStrategy);
-        Assert.Equal(304, retryStrategy.MaxRetryCount);
+        Assert.Equal(new WorkaroundToReadProtectedField(context).MaxRetryCount, retryStrategy.MaxRetryCount);
 
 #pragma warning restore EF1001 // Internal EF Core API usage.
     }
+
+    [Fact]
+    public void EnrichCanConfigureDbContextOptions()
```

Comment:
(nit) Why are these tests here, and not in `EnrichNpgsqlTests`?

Reply:
The class was named `AspireEFPostgreSqlExtensionsTests.cs` which is where they are. Didn't think we would do a test class for a specific method.

Reply:
Didn't realize you meant the one that just got created, moved them.